drivers/remote/api/api.go

```diff
@@ -24,7 +24,8 @@ func (r *Response) GetError() string {
 // GetCapabilityResponse is the response of GetCapability request
 type GetCapabilityResponse struct {
 	Response
-	Scope string
+	Scope     string
+	Multihost bool
```

Comment:
Its better if this is NOT a bool. For existing global scoped plugins, that is not implementing this new field yet will be incorrectly interpreted as false. So for drivers that doesnt set any value should default to its Scope value.
Hence, the flag name will be more like `ConnectivityScope` and string. It can accept `local` & `global`  (Just like the `Scope` field).